### PR TITLE
Avoid declaring preloadedWasm/prelaodedAudios/preloadedWasm on Module

### PR DESCRIPTION
**Pull request by sbc100**
These three globals were declared on the module using string names which                      
made them unfriendly to DCE.  In the past this may have been necessary,  
possibly due to a previous interaction with file-packager code.

### CODE REVIEW

**Comment by sbc100** [comment]
Looking at the history if these variables they go back to a97aab0522858f3e24ab96fe93e7be9e2b976878.. it looks like they were never intended to part of the incoming API

File: emcc.py [hunk 1]
```diff
@@ -147,7 +147,10 @@
 # These symbol names are allowed in INCOMING_MODULE_JS_API but are not part of the
 # default set.
 EXTRA_INCOMING_JS_API = [
-  'fetchSettings'
+  'fetchSettings',
+  'preloadedImages',
+  'preloadedAudios',
+  'preloadedWasm',
```
Comment (kripken) [inline emcc.py:153]: Do we even need this? I can't actually remember a use case for it.

**Review by kripken — COMMENTED** [review]
Hmm, I think these were on Module because file packager output code can access them... but reading that now I do not see any such access. In any case that path is well-tested so if we are missing something tests will complain.

File: emcc.py (hunk 1 above)
Comment (kripken) [reply emcc.py:153]: I mean, why would code set `Module.preloadedImages` on startup?

Comment (sbc100) [reply emcc.py:153]: I wasn't sure if that was valid.. but its looks like is at best unlikely.. and maybe completely invalid.  So I can just remove.

I agree these looks like they date back to a time when the file_packager output would use them..

Comment (kripken) [reply emcc.py:153]: Maybe it's safer to leave it for now. As it is not in the main list of INCOMING_JS it has no cost basically.

**Comment by sbc100** [comment]
Simplified and updated PR description

**Review by kripken — APPROVED** [review]
lgtm either way with the INCOMING_JS part. But if we want to remove it here, please mention that in the changelog, as maybe there is a use case for it that we are missing.

**Comment by kripken** [comment]
I guess someone could do preloading themselves and add the results to those maps. But adding a preload plugin would be the better way to do that anyhow, on `Module["preloadPlugins"]`. We could maybe to that if we want to remove the incoming part here.

**Comment by sbc100** [comment]
> lgtm either way with the INCOMING_JS part. But if we want to remove it here, please mention that in the changelog, as maybe there is a use case for it that we are missing.

I verified that it was not possible to set these on the incoming module.  Due to the location of the old code these lines would clobber anything incoming:

```
Module["preloadedImages"] = {}; // maps url to image data                                              
Module["preloadedAudios"] = {}; // maps url to audio data     
```

Thus I think no changelog entry needed.

**Comment by kripken** [comment]
Oh good, even better!

**Pull request merged**